app/src/main/java/com/taskruler/MainActivity.kt

```diff
@@ -54,6 +54,8 @@ fun Greeting(name: String) {
         Button(onClick = { /*TODO*/ })
         {Text(text = "Task Timed")}
 
+        Button(onClick = { /*TODO*/ })
+        {Text(text = "Start Timer")}
```

Comment:
I moved this button from the bottom of the page to the top to keep everything grouped together. This is a best practice according to https://kotlinlang.org/docs/coding-conventions.html#method-calls

Reply:
That's going to change the layout, which is a design decision... not necessarily tech debt reduction.